curation_concerns-models/app/models/concerns/curation_concerns/solr_document_behavior.rb

```diff
@@ -41,7 +41,7 @@ def human_readable_type
     end
 
     def representative_id
-      Array(self[Solrizer.solr_name('representative', :symbol)]).first
+      Array(self[Solrizer.solr_name('hasRelatedImage', :symbol)]).first
     end
```

Comment:
Why is this `hasRelated` image and not `hasRelatedMediaFragment` since it is for `representative_id` and not `thumbnail_id`?